### ISSUE TITLE
Add recursive 'Past as hard link(s)' script?

### ISSUE THREAD

**Issue by ak-42** [cfgnunes/nautilus-scripts#16]
An excellent project, thank you.

Could you please consider adding an option to recursively paste copied selection as hard links?

If the user selected a folder, it would recursively replicate the folder structure, and populate it with the hard links to files from the source and if the user selected one or more files, it would simply do a 1:1 hard link.
There should probably also be errors to prevent users trying to cross disk boundary, since that's not possible as hard links. Maybe "Would you like to paste a symlink instead?" confirmation?

Basically, a way to graphically invoke `cp -al`.

**Comment by cfgnunes** [comment]
Hi @ak-42  ! Thanks for the feature request!

I have implemented this feature now. It is in the 'main' branch. Could you test?
Thanks for your time!

The shortcut for "Paste as hard link" is "<Control><Alt>V"

**Comment by ak-42** [comment]
Hi @cfgnunes. Thank you for such a prompt response.

Did only a few tests, and it seems to be working well. Didn't notice any unexpected behaviour. I will make sure to file an issue if I do see any in the future after using the solution a bit more.

Many thanks for your efforts.
